site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-list.vm

```diff
@@ -270,12 +270,16 @@
 								#end
 							</td>
 							<td headers="title">
+								#if (!$site.isPublished() && !$service.allowUpdateSite($site.Id))
```

Comment:
Was mentioned on the team call by @jmarianolujan that you propbably want to be checking `site.visit.upnp` instead.